test/mysql.discover.test.js

```diff
@@ -74,7 +75,8 @@ describe('discoverModels', function () {
   });
 
   describe('Discover models excluding views', function () {
-    it('should return an array of only tables', function (done) {
+    // TODO: this test assumes the current user owns the tables
+    xit('should return an array of only tables', function (done) {
```

Comment:
`xit` is an alias for `it.skip`.. I guess it's only obvious if you're familiar with the convention. I'll change it to be more obvious.